content/post/announcing-pga.md

```diff
@@ -0,0 +1,129 @@
+---
+author: campoy
+date: 2018-06-05
+title: "Announcing Public Git Archive"
+image: /post/announcing-pga/pga.png
+description: "Announcing Public Git Archive, the largest git repositories dataset in the world."
+categories: ["technical", "MLonCode"]
+---
+
+Last week we had the honor of participating at [MSR'18](https://2018.msrconf.org/),
+where two of the members of our team, [Vadim Markovtsev](https://twitter.com/vadimlearning)
+and [Waren Long](https://twitter.com/warenlg), presented the
+[research paper](https://arxiv.org/abs/1803.10144) they wrote on our latest dataset:
+`Public Git Archive`.
+
+`Public Git Archive` is the result of months of effort curating a dataset suitable for
+training Machine Learning on Source Code (aka MLonCode) models.
+
+## Dataset Contents
+
+The dataset contains 3TB of repositories from GitHub ready to download.
+This includes all of the contents (git metadata and file contents) for all of the repositories
+on GitHub with 50 or more stars.
+The list of repositories was obtained from [the GHTorrent project](http://ghtorrent.org/),
+which provides information about GitHub repositories but unfortunately doesn't include the file contents.
```

Comment:
We took the GHTorrent snapshot of 01.01.2018 - they are published monthly

Reply:
fixed